src/main/java/ani/beautymarathon/service/MeasurementService.java

```diff
@@ -125,6 +134,82 @@ public Page<MoMeasurement> getCascadeOfAllMeasurements(MoMeasurementFilter filte
         }
     }
 
+    public WkMeasurement getWkById(long id) {
+        return wkMeasurementRepository.findById(id)
+                .orElseThrow(() -> new EntityNotFoundException("WkMeasurement with id " + id + " not found"));
+    }
+
+    public WkMeasurement updateWkStatus(long id, ClosedState closedState) {
+        final WkMeasurement wkMeasurement = getWkById(id);
+        wkMeasurement.setClosedState(closedState);
+
+        final WkMeasurement updated = wkMeasurementRepository.save(wkMeasurement);
```

Comment:
It'll be better to call this value such name as updatedWkMeas, updatedWkMeasurement, updWkMs for meaning of the value name